internal/scripts/build_image.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -euo pipefail
+
```

Comment:
bash has a builtin optarg parser which would be nicer than purely positional parameters.

Also add checks that all the input files actually exist.